### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by alexAubin** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    type = "group"
+    default = "visitors"
+
+    [install.language]
+    ask.en = "Choose the application language"
+    ask.fr = "Choisissez la langue de l'application"
+    type = "string"
+    choices = ["fr", "en"]
+    default = "fr"
+
```
Comment (alexAubin) [inline manifest.toml:45]:
```suggestion
```

Can't see where that "language" question is supposed to be used ...

**Comment by ericgaspar** [comment]
!testme

**Comment by Gredin67** [comment]
@ericgaspar @alexAubin on which branch should I start if I want to help ?

**Comment by ericgaspar** [comment]
> @ericgaspar @alexAubin on which branch should I start if I want to help ?

this one.

**Comment by Gredin67** [comment]
https://paste.yunohost.org/raw/ediqobusim
I don't get what is the error.

**Comment by alexAubin** [comment]
```
2024-02-08 01:56:22,792: DEBUG - + sudo -u fab-manager env RAILS_ENV=production PATH=/opt/rbenv/versions/fab-manager/bin:/opt/node_n/n/versions/node/18/bin:/opt/node_n/bin:/opt/rbenv/shims:/opt/rbenv/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so bin/bundle exec rake db:migrate
2024-02-08 01:56:55,258: DEBUG - rake aborted!
2024-02-08 01:56:55,260: DEBUG - KeyError: key not found: "DEFAULT_HOST"
2024-02-08 01:56:55,260: DEBUG - /var/www/fab-manager/config/initializers/sentry.rb:31:in `fetch'
2024-02-08 01:56:55,261: DEBUG - /var/www/fab-manager/config/initializers/sentry.rb:31:in `block in <main>'
2024-02-08 01:56:55,262: DEBUG - /var/www/fab-manager/config/initializers/sentry.rb:30:in `<main>'
2024-02-08 01:56:55,263: DEBUG - /var/www/fab-manager/config/environment.rb:5:in `<main>'
2024-02-08 01:56:55,263: DEBUG - bin/bundle:3:in `load'
2024-02-08 01:56:55,264: DEBUG - bin/bundle:3:in `<main>'
2024-02-08 01:56:55,264: DEBUG - Tasks: TOP => db:migrate => db:load_config => environment
```

**Comment by Gredin67** [comment]
I don't get it is defined here :
https://github.com/YunoHost-Apps/fab-manager_ynh/blob/version-2/conf/secrets.yml#L3C3-L3C29
And config file initialized here :

**Comment by Gredin67** [comment]
https://github.com/RailsApps/rails-stripe-membership-saas/issues/143

**Comment by Gredin67** [comment]
!testme

**Pull request closed without merging**